segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/metadata/Constants.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.metadata;
+
+/**
+ * Class that defines various constants.
+ */
+final class Constants {
```

Comment:
The name and description of this class is too generic. Please, refactor its name to its actual scope, like `StorageConstants` or `ChunkedStorageConstants` or something of the sort. Note that if we lookup for "Constants" in the IDE, it will take us more effort to understand what this class is used for, as the name itself is not helping.

Reply:
done.